src/crepe/system/AnimatorSystem.cpp

```diff
@@ -16,6 +16,7 @@ void AnimatorSystem::update() {
 	uint64_t tick = SDLContext::get_instance().get_ticks();
 	for (Animator & a : animations) {
 		if (a.active) {
+			// (10 frames per second)
```

Comment:
Mijn vraag is als gameprogrammeur. Als ik een poppetje sneller laat lopen wil ik dat de animatie sneller afspeelt is dit mogelijk?

Ik wil bijvoorbeeld met speed 10 (niet per frames want frames en physics zijn niet verbonden) dat het poppetje realistische beweegt is dit mogelijk?

Als ik dit wil moet ik dan in de script on update kijken hoever ik ben verplaats?
Ik wil dan een andere sprite in de spritesheet kiezen.
Is dit mogelijk of moeten dit allemaal losse sprites zijn?

Is het hiernaast ook nog mogelijk om ergens aan te geven hoelang een sprite van de animatie zichtbaar moet zijn?

Reply:
dit kan allemaal nog niet